services/api/src/configuration/index.ts

```diff
@@ -87,20 +87,20 @@ export const Configuration = {
     return env.get('WEB_URL').default('http://localhost:3000').asUrlString()
   },
 
+  get pinataApiKey() {
+    return env.get('PINATA_API_KEY').default('').asString()
+  },
+
+  get pinataApiSecret() {
+    return env.get('PINATA_API_SECRET').default('').asString()
+  },
+
   get sendgridApiKey() {
-    return env
-      .get('SENDGRID_API_KEY')
-      .default(
-        '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'
-      )
-      .asString()
+    return env.get('SENDGRID_API_KEY').asString()
```

Comment:
Sendgrid kindly asked me to remove these, even though they're just test things. If you're using these creds locally, they won't work anymore as I've decommissioned the key.

Reply:
This works. Could also default this to an empty string. 

```suggestion
    return env.get('SENDGRID_API_KEY').default('').asString()
```